.github/workflows/release.yml

```
@@ -0,0 +1,97 @@
+          context: ./${{ matrix.module }}
+          file: ./${{ matrix.module }}/Dockerfile
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ matrix.module }}:${{ env.VERSION }}
+
+  release-frontend:
+    name: Release Frontend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract version from branch name
+        run: echo "VERSION=${GITHUB_REF#refs/heads/release/}" >> $GITHUB_ENV
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:1:53: Double quote to prevent globbing and word splitting

(shellcheck)